extension/popup/popupView.jsx

```diff
@@ -890,11 +912,34 @@ const SearchResultsContent = ({
     </div>
   );
 
+  const MusicCard = () => (
```

Comment:
This should be called MusicSettingCard, since it's more specific than just music.